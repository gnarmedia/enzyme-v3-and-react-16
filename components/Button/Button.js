/**
 * @file Button component
 */

import React from 'react'
import PropTypes from 'prop-types'

const Button = ({ children }) => (
  <button type="button" className="button is-block is-info is-large is-fullwidth">{children}</button>
)

Button.propTypes = {
  children: PropTypes.node.isRequired
}

export default Button