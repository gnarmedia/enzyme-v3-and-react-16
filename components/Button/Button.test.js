import React from 'react'
import { shallow } from 'enzyme'

import Button from './Button'

describe('Button', () => {
  it('should render the Button component', () => {
    const wrapper = shallow(<Button>click me</Button>)

    expect(wrapper).toMatchSnapshot()
  })
})