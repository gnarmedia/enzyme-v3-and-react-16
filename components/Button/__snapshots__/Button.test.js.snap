// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render the Button component 1`] = `
<button
  className="button is-block is-info is-large is-fullwidth"
  type="button"
>
  click me
</button>
`;
